pkg/webui/components/input/index.js

```diff
@@ -150,8 +152,13 @@ class Input extends React.Component {
     const { focus } = this.state
 
     let Component = component
+    let inputStyle
     if (type === 'byte') {
       Component = ByteInput
+      const { max } = this.props
+      if (max && (inputWidth === 'auto' || max > 16)) {
```

Comment:
```suggestion
      if (max && (inputWidth === 'auto')) {
```

I don't think we need that, we should rather make sure that we set `inputWidth` actively.